README.md

```diff
@@ -2,16 +2,16 @@
  The GitHub API is documented here: https://docs.github.com/en/rest
 
  We will be connecting to this API to create a version of 'github'.
- You should clone this repo and in terminal type sbt run, in your browser on localhost:9000 you should see a page with the title "Welcome to your version of GitHub".
+ //You should clone this repo and in terminal type sbt run, in your browser on localhost:9000 you should see a page with the title "Welcome to your version of GitHub".
  
-  For package structure use standard: view (html templates), controller (API endpoint entry), service (Orchestration, should be light weight seeing as    we're only calling one endpoint), connector (To speak to api.github), model (Domain model data representation).
+  //For package structure use standard: view (html templates), controller (API endpoint entry), service (Orchestration, should be light weight seeing as    we're only calling one endpoint), connector (To speak to api.github), model (Domain model data representation).
  Using the GitHub API it’s a lot easier if upfront you start to use your own git credentials for authorisation (to avoid rate limits) -    https://developer.github.com/v3/auth/#basic-authentication (no need to worry about authorisation at the moment you can come back to this once you get to task 9 as authorisation is needed when you start creating and deleting in repositories on the github api)
 
- When writing your code only have working code on your main branch and for each new feature create a branch to create this. When this new feature is done and all the code is working create a pull request to merge your branch into main.
+ //When writing your code only have working code on your main branch and for each new feature create a branch to create this. When this new feature is done and all the code is working create a pull request to merge your branch into main.
  
 ## Task 1.
  Your mongodb should contain users which will have the following information: username, date account created, location, number of followers, number following.
-    Look at the return of a user from the GitHub API to help you make your model of a user. Try inputting your GitHub username into the url to see: https://api.github.com/users/{yourUsername}
+    Look at the return of a user from the GitHub API to help you make your model of a user. //Try inputting your GitHub username into the url to see: https://api.github.com/users/{yourUsername}
```

Comment:
Were we supposed to add controller, service and connector methods by this point? Or are they provided?